tutorial_app/middleware.py

```diff
@@ -4,6 +4,8 @@ class CustomAuthMiddleware(MiddlewareMixin):
     """
     Pass token from header to authorization
     """
+    # I think there will be an option in the JWT library to specify the 
```

Comment:
Look at this https://github.com/davesque/django-rest-framework-simplejwt/blob/master/rest_framework_simplejwt/authentication.py#L46.
I think they are getting it hardcoded instead of any variable.

Reply:
I see!

Reply:
Bad implentation by the library

Reply:
I have created an issue there.